doc/userDocs/usersGuide.md

```diff
@@ -94,9 +94,11 @@ Weathervane requires at least one existing Kubernetes cluster and a client syste
 - Kubernetes cluster:
     - You must have a kubeconfig file with [credentials](#credentials) for the cluster.
     - There must be at least one [StorageClass](#storageclass) defined on the cluster.
+        - A StorageClass Provisioner might have prerequisites and configuration [steps](#storageclass-prereq).
 - Client system:
     - Configure a workstation (MacOS or Linux) or a Linux VM with the required software:
         - Perl 5 (https://www.perl.org/get.html)
+        	- The module for Term::ReadKey should be installed.
```

Comment:
You should also include the command for this, e.g.:

The module for Term::ReadKey should be installed.  This can be achieved with the command `cpan Term::ReadKey`

Reply:
The cpan command itself may not be sufficient because installing this module has other prerequisites, such as make and gcc.

Reply:
That is unfortunate. It would be good if we could remove the dependency on Term::ReadKey, but that should be a separate issue.  You might as well leave your text as-is for now.